lms/templates/widgets/segment-io-footer.html

```diff
@@ -0,0 +1,14 @@
+% if settings.LMS_SEGMENT_KEY:
+    <!-- begin segment footer -->
```

Comment:
Good point. They could be the same except for the segment key name (they could both use the CMS implementation). I guess it isn't worth it.